providers/src/airflow/providers/fab/www/templates/appbuilder/navbar.html

```diff
@@ -0,0 +1,53 @@
+{#
```

Comment:
Copy of `airflow/www/templates/appbuilder/navbar.html`. I removed the right part of the header (containing user information). But maybe we want to keep it?

Reply:
Keeping it simple sounds good to me.